### DISCUSSION TITLE
.NET 8 Support?

### DISCUSSION THREAD

**Discussion by niclasrothman** [havit/Havit.Blazor#624] [Q&A] [answered]
Hi there, first thanks for a great library, it's been really useful to us!
Any plans for supporting upcoming .NET 8?

Regards Niclas

**Comment by hakenr** [comment]
.NET 8 is fully supported.
Did you encounter any specific issues?
Are you asking for any specific features coming with .NET 8?

**Reply by niclasrothman** [reply, marked as answer]
Hi Haken!
At the moment we are just preparing to migrate our application and I just wanted to know the status around Havit and .net 8, 
no issues found!

Regards Niclas
